src/map/ai/helpers/targetfind.cpp

```diff
@@ -267,6 +267,34 @@ void CTargetFind::addAllInEnmityList()
     }
 }
 
+void CTargetFind::addAllInRange(CBattleEntity* PTarget, float radius, uint8 allegiance)
+{
+    m_radius = radius;
+    m_PRadiusAround = &(m_PBattleEntity->loc.p);
+
+    // TODO: Use spawnlists instead of ForEachCharInstance and ForEachMobInstance
```

Comment:
I think these are OK for now given the scope of this PR, but afterwards, it would be good to make up some kind of utility that allows you to filter by allegiance, instead of this or my plan of iterating over all the spawn lists

Reply:
I guess this can be looked at later on, as of right now it does a good job as is.